### PR TITLE
Add new webhooks

### PR DESCRIPTION
**Pull request by szekelyzol**
Changes are for [this Asana task](https://app.asana.com/0/1205634133195403/1208756299627438).

**Summary**

- Added the new webhooks: `video.caption.generated` and `video.summary.generated`
- Updated the Create Webhook endpoint description, request schema, and response schema to list the webhook events as enums for the `events` field
- Added link to Transcript guide in the description for the `generationMode` attribute

### CODE REVIEW

File: oas_apivideo.yaml [hunk 1]
```diff
@@ -17458,6 +17458,110 @@ x-webhooks:
...
+                  type: string
+                  description: The name of the webhook event that occurred.
+                  example: video.caption.generated
+                emittedAt:
+                  description: Returns the date-time when the webhook event occurred.
+                  type: string
+                  format: date-time
+                  example: '2024-08-151T10:18:47+00:00'
+                videoId:
+                  description: The ID of the video where the caption was generated.
```
Comment (mioara-api-video) [inline oas_apivideo.yaml:17501]: I'd say "The ID of the video for which the caption was generated." I think!

File: oas_apivideo.yaml [hunk 2]
```diff
@@ -17458,6 +17458,110 @@ x-webhooks:
...
+                  type: string
+                  description: The name of the webhook event that occurred.
+                  example: video.caption.generated
+                emittedAt:
+                  description: Returns the date-time when the webhook event occurred.
+                  type: string
+                  format: date-time
+                  example: '2024-08-151T10:18:47+00:00'
+                videoId:
+                  description: The ID of the video where the summary was generated.
```
Comment (mioara-api-video) [inline oas_apivideo.yaml:17558]: I'd say "The ID of the video for which the summary was generated." I think!

**Review by olivier-lando — APPROVED** [review]

File: oas_apivideo.yaml (hunk 1 above)
Comment (szekelyzol) [reply oas_apivideo.yaml]: Fixed ✅

File: oas_apivideo.yaml (hunk 2 above)
Comment (szekelyzol) [reply oas_apivideo.yaml]: Fixed ✅

**Pull request merged**